### DISCUSSION TITLE
company manual: minor suggestion

### DISCUSSION THREAD

**Discussion by mandarm** [company-mode/company-mode#1500] [General]
Not really sure what's the best place for this suggestion..

The [Candidates Post-Processing](https://company-mode.github.io/manual/Backends.html#Candidates-Post_002dProcessing) section of the manual uses ```delete-consecutive-dups``` as an example that could be added to ```company-transformers```. OTOH, the docstring for ```company-transformers``` says

The first one [first function on the ```company-transformers``` list] gets passed the list of candidates, already sorted and without duplicates.

It appears then, that ```delete-consecutive-dups``` would not be a useful function to add to ```company-transformers```. If this understanding is correct, perhaps the manual could be modified suitably to avoid any confusion?

**Comment by yugaego** [comment]
Thanks for bringing this up. It looks like we've got a room for a slight improvement here. I migrated this discussion to the issue #1502 .
